crawler/store/spar.py

```diff
@@ -170,10 +170,6 @@ def parse_store_from_filename(self, filename: str) -> Optional[Store]:
         )
         return store
 
-    def parse_csv_row(self, row: dict) -> Product:
```

Comment:
The original reason for this is to support both with " (EUR)" suffix and without it, to be able to fetch old data.

Now we're sufficiently removed in time from that change so I guess it's okay, and it does make the system cleaner.